src/stores/DomainStore.ts

```diff
@@ -26,7 +26,7 @@ export class DomainStore {
   }
 
   async fetchData() {
-    const response = await fetch('/api/domain?select=*,topic_of_research(id,unique_label)');
+    const response = await fetch(`/api/domain?select=*,topic_of_research(id,unique_label)`);
```

Comment:
Not sure why this change is here

Reply:
I noticed that you used \` with fetch in most of the cases fetch(\`...\`), so I updated this one.

Reply:
In this case it made me suspect you might have forgotten to interpolate some parameters

Reply:
I see, I was not aware of the difference in the syntax, so I will revert the change.

Reply:
Done